### ISSUE TITLE
Nightstalker Aghanim Slow only Applies Proper Slow Duration on 1 target

### ISSUE THREAD

**Issue by girvinjunod** [ValveSoftware/Dota2-Gameplay#15107]
### Ability name

Void

### Description

TLDR: Nightstalker's aghs upgrade for his 1st skill (Void) is supposed to make void into an aoe ability with 1s stun and 4s slow on nighttime. This bug causes aghs void to only apply that nighttime slow on 1 target even though it's an aoe spell.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/68438200/2db80be3-a4ba-47f7-b056-ba790b283ccd)

Void on night is supposed to slow the enemy for 4 seconds compared to 1.25 second on daytime

However, on testing with aghs void only applies 4 second slow on 1 enemy and only applies 1 second slow to the rest, not even the supposed 1.25 second slow on daytime. As you can see on the log, the void debuff times out exactly with the 1 second duration stun. The target chosen by void for the 4s slow is random from my testing.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/68438200/3c3e9ece-4c71-4b9c-be1e-1e2aba94332b)

The mechanic for aghs is also pretty wonky. This bug persists even during nighttime or during ultimate. At daytime, the aghs only applies the 4s slow on one target only if Nightstalker is in the buff zone on cast, otherwise all target will only get the proper daytime 1.25s slow.

All in all it's a pretty huge bug considering void slow is both ms speed and attack speed slow.

### Example Match ID (and possibly Timestamp)

-
